overview.md

```diff
@@ -44,23 +50,26 @@ Scope Frameworks define the rules that SubDAOs must adhere to. Scope Frameworks
 
 Scope Frameworks limit the governance overhead by ensuring that Maker Core only focuses on high-level decisions. They also encourage innovation among SubDAOs without allowing them to take excessive risks that may damage Maker Core. 
 
-### [Decentalized Voter Committees](maker-core/dvc.md)
+### [Constitutional Voter Committees](maker-core/cvc.md)
```

Comment:
Probably needs more detail. In practice this is more than just a name-change. The constitutional voter committees must commit to maintaining endgame.